### PR TITLE
Inline get_file_owner_user and get_file_owner_group

### PR DESCRIPTION
**Pull request by eagletmt**
I've created a small mrbgem for this patch.
https://github.com/eagletmt/mruby-etc

### CODE REVIEW

File: mrbgem.rake [hunk 1]
```diff
@@ -33,4 +33,5 @@ MRuby::Gem::Specification.new('mitamae') do |spec|
 
   spec.add_dependency 'mruby-erb',       github: 'k0kubun/mruby-erb', checksum_hash: '978257e478633542c440c9248e8cdf33c5ad2074'
   spec.add_dependency 'mruby-io',        github: 'k0kubun/mruby-io',  checksum_hash: '6cb5d157341ceec8f5818ce0000fa99920258c11'
+  spec.add_dependency 'mruby-etc',       github: 'eagletmt/mruby-etc',  checksum_hash: 'v0.1.0'
```
Comment (k0kubun) [inline mrbgem.rake]: It's very helpful to have tag. :+1: 
But I've never seen mrbgem that manages releases by a tag other than mruby-etc...

File: mrblib/mitamae/inline_backends/file_backend.rb [hunk 2]
```diff
@@ -19,6 +19,22 @@ def get_file_mode(path)
       rescue => e
         Specinfra::CommandResult.new(stdout: '', stderr: e.message, exit_status: 1)
       end
+
+      def get_file_owner_user(path)
+        uid = File.stat(path).uid
+        passwd = Etc.getpwuid(uid)
+        Specinfra::CommandResult.new(stdout: passwd.name, stderr: '', exit_status: 0)
+      rescue => e
+        Specinfra::CommandResult.new(stdout: '', stderr: e.message, exit_status: 1)
```
Comment (k0kubun) [inline mrblib/mitamae/inline_backends/file_backend.rb]: By the way, I think InlineBackend is not related to Specinfra and it should have a different struct like Specinfra::CommandResult.  WDYT?

Comment (k0kubun) [inline mrblib/mitamae/inline_backends/file_backend.rb]: And `e.message` isn't stderr and should be logged as such ideally.

**Comment by k0kubun** [comment]
I added some comments but this change is acceptable. Thank you!

File: mrblib/mitamae/inline_backends/file_backend.rb (hunk 2 above)
Comment (eagletmt) [inline mrblib/mitamae/inline_backends/file_backend.rb]: +1 for adding a different class which acts like Specinfra::CommandResult.

Previously, `run_command` case raises an error when the command fails, which is different from this patch.
I think raising the original exception is better than logging exceptions and it's more similar to the previous behavior.
I'm preparing a patch for it.

**Pull request merged**